test/opera-rtc-s1-browse/opera-rtc-s1-browse_Regression.ipynb

```diff
@@ -0,0 +1,91 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "24ad02a3-f52b-4e10-8654-e28d3e998c59",
+   "metadata": {
+    "tags": [
+     "parameters"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "harmony_host_url = 'https://harmony.uat.earthdata.nasa.gov'"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "9692ec8e-89e4-4742-9b11-57a8644cd770",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from pathlib import Path\n",
+    "from tempfile import TemporaryDirectory\n",
+    "\n",
+    "import harmony\n",
+    "\n",
+    "\n",
+    "environments = {\n",
+    "    'http://localhost:3000': harmony.Environment.LOCAL,\n",
+    "    'https://harmony.sit.earthdata.nasa.gov': harmony.Environment.SIT,\n",
+    "    'https://harmony.uat.earthdata.nasa.gov': harmony.Environment.UAT,\n",
+    "    'https://harmony.earthdata.nasa.gov': harmony.Environment.PROD,\n",
+    "}\n",
+    "assert harmony_host_url in environments\n",
+    "harmony_client = harmony.Client(env=environments[harmony_host_url])\n",
+    "\n",
+    "request = harmony.Request(\n",
+    "    collection=harmony.Collection(id='OPERA_L2_RTC-S1_V1'),\n",
+    "    granule_name=[\n",
+    "        'OPERA_L2_RTC-S1_T035-073251-IW2_20240512T020817Z_20240512T122756Z_S1A_30_v1.0'\n",
+    "    ],\n",
```

Comment:
This is probably the nicer way to make a request run across multiple environments rather than using different concept IDs for local/SIT/UAT versus production. Nice.

Reply:
Exactly. Collection short name and granule UR are almost always my preferred identifiers over concept IDs. Concept IDs are devoid of human-readable context, are generated by CMR, and differ from environment to environment. Short name and granule UR include human readable details, are assigned by the DAAC, and are (usually) the same from environment to environment.